tools/quantification/.shed.yml

```diff
@@ -0,0 +1,6 @@
+owner: perssond
```

Comment:
is this the correct owner?

Reply:
Yes, we weren't able to change the toolshed owner, so still with this.